luigi_tools/task.py

```diff
@@ -52,6 +53,17 @@ def __init__(self, *args, **kwargs):
         if recursive_check(self):
             apply_over_outputs(self, target_remove)
 
+        if isinstance(self, luigi.WrapperTask) and self.rerun:
+
+            def recursive_target_remove(target, *args, **kwargs):
+                outputs = getattr(target, "output", lambda: [])()
+                apply_over_luigi_iterable(outputs, target_remove)
```

Comment:
Is this needed? WrapperTask() has not output() and its complete() depends only on its requires().

Reply:
I want to apply the same function to all tasks of the dependency tree, which can contain both Task and WrapperTask classes. So for each dependency I clear both the outputs of the task and the ones of the dependencies of this task.